src/Modules/Finance/FinanceLayout.tsx

```diff
@@ -3,13 +3,9 @@ import {Header} from './Header/Header';
 
 import styles from './FinanceLayout.module.css';
 
-export class FinanceLayout extends React.Component {
-    public render(): React.ReactNode {
-        return (
-            <div className={styles.FinanceLayout}>
-                <h1>Finance module here</h1>
-                <Header />
-            </div>
-        );
-    }
-}
+export const FinanceLayout = () => (
```

Comment:
Should we add some TS stuff to this line?

Reply:
I would say in the future we will  do this, I mean when some props will be specified.